source/MRMesh/MRPrecisePredicates2.h

```diff
@@ -11,6 +11,17 @@ namespace MR
 /// \ingroup MathGroup
 /// \{
 
+struct PreciseVertCoord
+{
+    VertId id;   ///< unique id of the vertex (in both contours)
+    int pt;      ///< coordinate
+};
```

Comment:
First, we will have a conflict here with MRPrecisePredicates3.h:
```
struct PreciseVertCoords
{
    VertId id;   ///< unique id of the vertex (in both meshes)
    Vector3i pt; ///< integer coordinates of the vertex
};
```
And second, why int and not Vector2i?